docker/nmslib/build-image.sh

```diff
@@ -0,0 +1,3 @@
+#!/usr/bin/env bash
```

Comment:
I dont think this file is necessary. What is the use case for it?

Reply:
It's only for development purposes to help build the images for testing. I was considering ignoring it but I figured it *may* be useful to anyone developing with the Docker setup. Should I ignore/untrack it?

Reply:
On second thought, it appears the build parameters can be specified in the `docker-compose.yml` and the resulting image can be [built with `docker-compose build`](https://docs.docker.com/compose/compose-file/compose-file-v3/#build). Perhaps these build paramters can be put in the `docker-compose.yml` instead?

Reply:
Yeah, I think that makes sense. If we are able to parametrize Dockerfile with env variables in docker-compose as well that would be awesome